internal/envtest/webhooks.go

```diff
@@ -38,8 +40,8 @@ const (
 )
 
 func initWebhookInstallOptions() envtest.WebhookInstallOptions {
-	validatingWebhooks := []client.Object{}
-	mutatingWebhooks := []client.Object{}
+	validatingWebhooks := []admissionv1.ValidatingWebhookConfiguration{}
+	mutatingWebhooks := []admissionv1.MutatingWebhookConfiguration{}
 
```

Comment:
This was necessary because WebhookInstallOptions now expects the correct types and not just client.Object